File: fstmerge/examples/Stack_python/Serializable/stack.py
import os
import sys
import re

gfoo = False

class Stack:
	items = llist()
	check = first
	check2 = second

	def push(self, item):
		self.items.append(item)

	serialVersionUID = 42

	def pop(self):
		if len(self.items) > 0: return self.items.pop()
		else: return None

File: fstmerge/examples/Stack/Size/Stack.java
import java.util.LinkedList;

public class Stack<T> {
	private LinkedList<T> items = new LinkedList<T>();
	public Stack() {
		int j = 0;
		int k = 0;
	}
	public void push(T item) {
		items.addFirst(item);
	}
	public int size() {
		return items.size();
	}
	public T pop() {
		int i = 0;
		if(items.size() > 0)
			return items.removeFirst();
		else
			return null;
	}
}